manager_test.go

```
@@ -106,70 +106,71 @@ func TestManager_AwaitFire(t *testing.T) {
 	assert.Equal(t, "Hi > 1 app.evt1 > 2 app.evt1 > 3 app.evt1", buf.String())
 
 	em.RemoveListener("app.*", l2)
 	assert.Len(t, em.ListenedNames(), 2)
-	em.On("app.*", event.ListenerFunc(func(e event.Event) error {
+	em.On("app.*", event.ListenerFunc[event.M](func(e event.Event[event.M]) error {
 		return fmt.Errorf("an error")
 	}))
 
 	buf.Reset()
-	err, e = em.Fire("app.evt1", nil)
+	err, e = em.Fire("app.evt1", event.M{})
```

golangci-lint (1.64.8)

ineffectual assignment to e

(ineffassign)